core/modules/views/includes/view.inc

```diff
@@ -310,12 +312,12 @@ class view {
   /**
    * Perform automatic updates when loading or importing a view.
    *
-   * Over time, some things about Views or Backdrop data has changed.
-   * this attempts to do some automatic updates that must happen
-   * to ensure older views will at least try to work.
+   * Over time, some things about Views or Backdrop data has changed. This
+   * attempts to do some automatic updates, that must happen to ensure older
+   * views will at least try to work.
    */
   function update() {
-    // When views are converted automatically the base_table should be renamed
+    // When views are converted automatically, the base_table should be renamed,
```

Comment:
No comma needed after 'renamed'. First comma possibly also not needed, but happy to leave this one.

Reply:
I don't agree, because otherwise one may interpret the sentence in different ways (it helps if you mentally replace "to" with "in order to"). Perhaps changing it to this makes more sense:

> To have a working query when views are converted automatically, the base_table should be renamed.